Emergency-Response-Demo/mobile-app
# add login view

to verify:

* check out the PR and install all the dependencies, and then ran `npm run start`
* You should see the home page
* Modify `mobile-services.json` file and add the following object to the `services` list:
   ```
   {
      "config": {
        "auth-server-url": "https://sso-user-sso.apps.demo-1762.open.redhat.com/auth",
        "confidential-port": 0,
        "public-client": true,
        "realm": "app1-realm",
        "resource": "app1-client",
        "ssl-required": "external"
      },
      "id": "fba3edde-d49a-11e9-be3a-025c402b324e",
      "name": "keycloak",
      "type": "keycloak",
      "url": "https://sso-user-sso.apps.demo-1762.open.redhat.com/auth"
    }
   ``` 
start the app again and you should see a login page.
